mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -1631,7 +1638,15 @@ public Source resolve(String href, String base) {
 
             final Element i18nElement = new Element("i18n");
             if (!target.contains("*") && !target.contains(",")) {
-                i18nElement.addContent(MCRTranslation.translate(target));
+                String translation;
+                if (target.contains(":")) {
+                    translation = new MessageFormat(
```

Comment:
Formating should be handled by MCRTranslation:
`final int i = target.indexOf(":");
        MCRTranslation.translate(target.substring(0, i), (Object[]) target.substring(i+1).split(":"));`

Reply:
Appears to work, thanks.